src/stars.css

```diff
@@ -0,0 +1,28 @@
+.stars--4 {
+  margin: 0;
+}
```

Comment:
use mix for external styles
check checklist examples
<img width="946" alt="image" src="https://user-images.githubusercontent.com/102411130/211768866-65a28ab6-e41c-4ce8-a33a-f30add3653cf.png">

Reply:
I understand what you mean, but I don't understand how it was all good to poke around